### PR TITLE
Ch api tests 130627013

### PR DESCRIPTION
**Pull request by karanjaE**

### CODE REVIEW

File: hc/api/tests/test_create_check.py [hunk 1]
```diff
@@ -49,21 +53,21 @@ def test_it_accepts_api_key_in_header(self):
...
+        self.assertEqual(r.status_code, 400, msg=r.json())
+        self.assertEqual(r.json()["error"], "wrong api_key", msg=r)
 
     def test_it_handles_invalid_json(self):
         ### Make the post request with invalid json data type
-        r = {'status_code': 400, 'error': "could not parse request body"} ### This is just a placeholder variable
-        self.assertEqual(r['status_code'], 400)
-        self.assertEqual(r["error"], "could not parse request body")
+        r = self.client.post(self.URL, 'abcdef', content_type="application/json")
+        self.assertEqual(r.status_code, 400, )
```
Comment (mfalade) [inline hc/api/tests/test_create_check.py:69]: Change this to

``` python
self.assertEqual(r.status_code, 400)
```

File: hc/api/tests/test_create_check.py [hunk 2]
```diff
@@ -49,21 +53,21 @@ def test_it_accepts_api_key_in_header(self):
...
+        self.assertEqual(r.status_code, 201, r)
 
     def test_it_handles_missing_request_body(self):
         ### Make the post request with a missing body and get the response
-        r = {'status_code': 400, 'error': "wrong api_key"} ### This is just a placeholder variable
-        self.assertEqual(r['status_code'], 400)
-        self.assertEqual(r["error"], "wrong api_key")
+        r = self.client.post(self.URL, content_type="application/json")
+        self.assertEqual(r.status_code, 400, msg=r.json())
+        self.assertEqual(r.json()["error"], "wrong api_key", msg=r)
```
Comment (mfalade) [inline hc/api/tests/test_create_check.py]: Remove the `msg` kwarg here.

File: hc/api/tests/test_list_checks.py [hunk 3]
```diff
@@ -1,5 +1,8 @@
-import json
+import datetime
```
Comment (mfalade) [inline hc/api/tests/test_list_checks.py:1]: Remove `datetime` import

File: hc/api/tests/test_create_check.py (hunk 2 above)
Comment (karanjaE) [reply hc/api/tests/test_create_check.py]: I have removed the kwarg

**Pull request closed without merging**